apostrophecms/apostrophe
# PRO-6225: nested errors

## Summary

Identify and mark server validation errors in the admin UI. This helps editors identify already existing data fields having validation errors when schema changes (e.g. optional fields becomes required). 

## What are the specific steps to test this change?

- Create nested in multiple arrays text field, mark it `required: false`
- Add the array data to a document but leave the text field empty
- Change the schema so that the text field is now required
- Edit the document, change its title and try to safe
- You should see error messages leading you all down to that field.

## What kind of change does this PR introduce?
*(Check at least one)*


## Make sure the PR fulfills these requirements:


If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**
